launch/master_slaves_pwm_rgb8.launch

```diff
@@ -0,0 +1,167 @@
+          file, and before dynamic_reconfigure. That means that any
+          (lingering) dynamic parameters from dynamic_reconfigure will
+          override these values, and that these will override parameters
+          from the .ini file.
+          See http://www.ros.org/wiki/ueye_cam for more details. -->
+
+     <param name="color_mode" type="str" value="rgb8" /> <!-- valid options: 'rgb8', 'mono8', 'bayer_rggb8' -->
+
+     <!-- WARNING: the following 4 parameters specify dimensions for camera's area of interest. Values for image_width and image_height that are smaller than your camera's maximum values will result in cropped field of view. For typical cases, one should modify values for sensor_scaling / binning / subsampling to downsample the resulting ROS image to the desired dimensions, without losing potential field of view. -->
+     <param name="image_width" type="int" value="640" />
+     <param name="image_height" type="int" value="480" />
+     <param name="image_top" type="int" value="-1" /> <!-- -1: center -->
+     <param name="image_left" type="int" value="-1" /> <!-- -1: center -->
+
+     <param name="subsampling" type="int" value="1" /> <!-- supported by only some UEye cameras -->
+     <param name="binning" type="int" value="1" /> <!-- supported by only some UEye cameras -->
+     <param name="sensor_scaling" type="double" value="1.0" /> <!-- supported by only some UEye cameras -->
+
+     <param name="auto_gain" type="bool" value="True" />
+     <param name="master_gain" type="int" value="0" />
+     <param name="red_gain" type="int" value="0" />
+     <param name="green_gain" type="int" value="1" />
+     <param name="blue_gain" type="int" value="16" />
+     <param name="gain_boost" type="bool" value="False" />
+
+     <param name="auto_exposure" type="bool" value="False" />
+     <param name="exposure" type="int" value="33" /> <!-- in ms -->
+
+     <param name="auto_white_balance" type="bool" value="True" />
+     <param name="white_balance_red_offset" type="int" value="0" />
+     <param name="white_balance_blue_offset" type="int" value="0" />
+
+     <param name="flash_delay" type="int" value="30000" /> <!-- in us -->
+     <param name="flash_duration" type="int" value="1000" /> <!-- in us -->
+
+     <param name="gpio2" type="int" value="4" /> <!-- 0 Input, 1 Output Low, 2 Output High, 3 Flash, 4 PWM, 5 Trigger input  -->
+     <param name="pwm_freq" type="double" value="1 " /> <!-- in hz, This will determine the framerate of all cameras -->
+     <param name="gpio1" type="int" value="5" /> <!-- 0 Input, 1 Output Low, 2 Output High, 3 Flash, 4 PWM, 5 Trigger input  -->
+
+     <param name="pixel_clock" type="int" value="30" />
+     </node>
+
+     <node pkg="nodelet" type="nodelet" name="ueye_cam_nodelet_slave1"
+          args="load ueye_cam/ueye_cam_nodelet nodelet_manager">
+     <param name="camera_name" type="str" value="master" /> <!-- == namespace for topics and services -->
```

Comment:
Since the camera name and topic are the same, ROS is going to throw an error about trying to advertise the same service multiple times.